tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/FallbackTest.java

```diff
@@ -81,17 +82,17 @@ public void testFallbackSuccess() {
             String result = fallbackClient.serviceA();
             Assert.assertTrue(result.contains("serviceA"),
                     "The message should be \"fallback for serviceA\"");
-        } catch (RuntimeException ex) {
-            Assert.fail("serviceA should not throw a RuntimeException in testFallbackSuccess");
+        } catch (TestException ex) {
+            Assert.fail("serviceA should not throw a RuntimeException in testFallbackSuccess", ex);
         }
         Assert.assertEquals(fallbackClient.getCounterForInvokingServiceA(), 2,
                 "The execution count should be 2 (1 retry + 1)");
         try {
             String result = fallbackClient.serviceB();
             Assert.assertTrue(result.contains("serviceB"),
                     "The message should be \"fallback for serviceB\"");
-        } catch (RuntimeException ex) {
-            Assert.fail("serviceB should not throw a RuntimeException in testFallbackSuccess");
+        } catch (TestException ex) {
+            Assert.fail("serviceB should not throw a RuntimeException in testFallbackSuccess", ex);
```

Comment:
The changes to the `catch` blocks in this test are good, but the following methods in other classes should be updated to match:

- `FallbackClient.nameService()`
- `FallbackOnlyClient.nameService()`
- `FallbackWithBeanClient.nameService()`